### PR TITLE
Add Dockerfile.rhel7

### PR DESCRIPTION
**Pull request by damemi**

### CODE REVIEW

File: Dockerfile.rhel7 [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.10 AS builder
+WORKDIR /go/src/github.com/openshift/secondary-scheduler-operator
+COPY . .
+
+RUN go build -o secondary-scheduler-operator ./cmd/cluster-kube-descheduler-operator
+
+FROM registry.ci.openshift.org/ocp/4.10:base
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+COPY --from=builder /go/src/github.com/openshift/secondary-scheduler-operator/secondary-scheduler-operator /
```
Comment (wangchen615) [inline Dockerfile.rhel7:9]: How about we removing the ENTRYPOINT and making it consistent with Dockerfile??

```
COPY --from=builder /go/src/github.com/openshift/secondary-scheduler-operator/secondary-scheduler-operator /usr/bin/
```

File: Dockerfile.rhel7 [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Upstream bundle and index images does not support versioning so
+# we need to copy a specific version under /manifests layout directly
+COPY --from=builder /go/src/github.com/openshift/secondary-scheduler-operator/manifests/4.9/* /manifests/
+
+ENTRYPOINT ["/secondary-scheduler-operator"]
```
Comment (wangchen615) [inline Dockerfile.rhel7:14]: And remove this line.

**Review by wangchen615 — CHANGES_REQUESTED** [review]

File: Dockerfile.rhel7 (hunk 1 above)
Comment (ingvagabund) [reply Dockerfile.rhel7:9]: +1, let's stick with the form we have in https://github.com/openshift/cluster-kube-descheduler-operator/blob/master/Dockerfile.rhel7 to keep both files in sync.

File: Dockerfile.rhel7 [hunk 3]
```diff
@@ -0,0 +1,20 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.10 AS builder
+WORKDIR /go/src/github.com/openshift/secondary-scheduler-operator
+COPY . .
+
+RUN go build -o secondary-scheduler-operator ./cmd/cluster-kube-descheduler-operator
```
Comment (soltysh) [inline Dockerfile.rhel7:5]: Any particular reason we're not using `make build`, I've noticed we're doing similar in descheduler-operator?

**Comment by damemi** [comment]
Addressed the comments, if it looks good feel free to merge

**Pull request merged**